### PR TITLE
feat(onboarding): update verification styles

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

Redesigns for verification screens

Figma: https://www.figma.com/file/NAp0jPTvldgl8AVajej3ns/Onboarding-Updates?node-id=1801%3A16368&t=fSX2mpcn9Zs1SVIO-0

#### Verification Start

| First Render | Learn More |
|:-----:|:-----:|
| Simulator Screen Shot - iPhone 13 - 2023-02-17 at 11 29 48 | Simulator Screen Shot - iPhone 13 - 2023-02-17 at 11 30 10 

[Video1](https://user-images.githubusercontent.com/8432644/219787477-12e9cddd-cc8a-4c60-bec7-ba9979ff7f53.mp4)

#### Verification Code Input

| Code Input | Code Help | Code Trying 
|:-----:|:-----:|:-----:|
| Simulator Screen Shot - iPhone 13 - 2023-02-17 at 11 32 33 | Simulator Screen Shot - iPhone 13 - 2023-02-17 at 11 32 43 | Simulator Screen Shot - iPhone 13 - 2023-02-17 at 11 33 02 | 

https://user-images.githubusercontent.com/8432644/219788470-87aacc11-b1a7-47ec-b718-2b30ba35037d.mp4

### Test plan

Unit tests

### Related issues

https://linear.app/valora/issue/ACT-606/update-phone-verification-screen

### CODE REVIEW

File: src/verify/VerificationStartScreen.tsx [hunk 1]
```diff
@@ -265,28 +256,13 @@ function VerificationStartScreen({
...
-        title={t('phoneVerificationScreen.skip.title')}
-        isVisible={showSkipDialog}
-        actionText={t('phoneVerificationScreen.skip.confirm')}
-        actionPress={onPressSkipConfirm}
-        secondaryActionPress={onPressSkipCancel}
-        secondaryActionText={t('phoneVerificationScreen.skip.cancel')}
-        testID="PhoneVerificationSkipDialog"
-      >
-        {t('phoneVerificationScreen.skip.body')}
-      </Dialog>
```
Comment (satish-ravi) [inline src/verify/VerificationStartScreen.tsx:278]: Just double checking, design / product were ok with dropping this confirm dialog?
Also, should we remove corresponding translations?

File: src/components/BottomSheet.tsx [hunk 2]
```diff
@@ -89,8 +95,8 @@ const styles = StyleSheet.create({
   },
   background: {
     position: 'absolute',
-    backgroundColor: colors.modalBackdrop,
-    opacity: 0.5,
+    backgroundColor: '#2E3338',
+    opacity: 0.25,
```
Comment (satish-ravi) [inline src/components/BottomSheet.tsx:99]: are these safe to change? These are used in places other than onboarding right?

File: src/verify/VerificationStartScreen.tsx (hunk 1 above)
Comment (jh2oman) [reply src/verify/VerificationStartScreen.tsx:278]: Yes I double checked with them in Dm's. Its okay to drop

File: src/components/BottomSheet.tsx (hunk 2 above)
Comment (jh2oman) [reply src/components/BottomSheet.tsx:99]: probably best not to. I'll refactor

File: locales/base/translation.json [hunk 3]
```diff
@@ -747,13 +747,13 @@
...
+    "title": "Connect your phone number to Valora",
+    "description": "This links your wallet address to your phone number so sending crypto is easy",
+    "startButtonLabel": "Connect",
     "learnMore": {
-      "buttonLabel": "Do I need to connect my phone?",
-      "title": "Do I need to connect my phone?",
-      "body": "Connecting your phone number will allow you to send money to friends in your address book and other phone numbers. \n \n If you choose to skip this step you won’t be able to access your contacts and you will only be able to send money via QR code or Account Address."
+      "buttonLabel": "Do I need to connect?",
+      "title": "Do I need to connect?",
+      "body": "Connecting your phone number will allow you to send money to friends in your address book and other phone numbers. \n \n If you choose to skip this step you won’t be able to access your contacts and you will only be able to send money via QR code or wallet address."
```
[the remark this answers is not in the document]
Comment (jh2oman) [reply locales/base/translation.json:756]: Yea they still make sense. Very small changes.

File: e2e/src/usecases/NewAccountPhoneVerification.js [hunk 4]
```diff
@@ -170,28 +170,13 @@ export default NewAccountPhoneVerification = () => {
...
-
-    // Assert PhoneVerificationSkipDialog modal actions are visible
-    await waitForElementId('PhoneVerificationSkipDialog/PrimaryAction')
-    await waitForElementId('PhoneVerificationSkipDialog/SecondaryAction')
 
     // Assert Back button is enabled
     let goBackButtonAttributes = await element(by.text('Go Back')).getAttributes()
     jestExpect(goBackButtonAttributes.enabled).toBe(true)
 
     // Tap 'Skip for now'
```
Comment (satish-ravi) [inline e2e/src/usecases/NewAccountPhoneVerification.js:178]:
```suggestion
    // Tap 'Skip'
```

**Review by satish-ravi — APPROVED** [review]

**Pull request merged**